### DISCUSSION TITLE
How to get the display working on a new map?

### DISCUSSION THREAD

**Discussion by wraethcsgo** [Linus4/csgoverview#55] [Q&A] [answered]
Hi, appreciate all the work you've put in. I guess you're working on a new version, but for now I was wondering if there was a quick fix to getting de_anubis working on csgoverview. With a radar jpg in the directory it is possible to get the display working and even see the players on the screen, but they all seem "off map" on the top left. Any way to address just that so we can start analyzing anubis demos?

Again, thanks for your product. It's amazing for a free app.

**Comment by Linus4** [comment, marked as answer]
Hi @wraethcsgo , thanks for reaching out! Unfortunately, this will have to be part of a larger rework.

The parser used to provide the metadata for the maps, 'so the players wouldn't seem to be off map'. But in [version 3](https://github.com/markus-wa/demoinfocs-golang/releases/tag/v3.0.0), which was released earlier this year, this functionality was deprecated and moved to [another project](https://github.com/saiko-tech/csgo-centrifuge).
So, in addition to upgrading to the new major version, csgoverview needs a new way to access map metadata. Maybe I'll find time over the holidays to work on it, but there's no quick way to address the problem. Once it's done, though, this kind of problem should be fixed for the forseeable future.
